test/features/schema_tests.js

```diff
@@ -11,6 +11,11 @@ var Ajv = require('ajv');
 describe('Responses should conform to the provided JSON schema of the responce', function() {
     var ajv = new Ajv({});
 
+    function getToday() {
+        const now = new Date();
+        return `${now.getUTCFullYear()}/${now.getUTCMonth() + 1}/${now.getUTCDate()}`;
```

Comment:
@gwicke This is a part of public API (`/feed/featured/{yyyy}/{mm}/{dd}`) so we can't change it any more.